docs/provider/infisical.md

```
@@ -779,6 +779,7 @@ The `secretsScope` configuration controls which secrets are accessible:
 | `secretsPath` | No | `/` | The base path for secrets retrieval |
 | `recursive` | No | `false` | When true, fetches secrets recursively from subfolders |
 | `expandSecretReferences` | No | `true` | When true, expands secret references (e.g., `${SECRET_NAME}`) |
+| `keepDuplicateKeys` | No | `false` | When true, keep duplicate keys by appending the secret path to it (e.g. `/path/to/secret:SECRET_NAME`)
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)